content/en/docs/tasks/access-application-cluster/web-ui-dashboard.md

```diff
@@ -34,12 +34,16 @@ You can access Dashboard using the kubectl command-line tool by running the foll
 kubectl proxy
 ```
 
-Kubectl will handle authentication with apiserver and make Dashboard available at http://localhost:8001/api/v1/namespaces/kube-system/services/https:kubernetes-dashboard:/proxy/.
+Kubectl will handle authentication with the apiserver and make the
+Dashboard UI available at ``http://localhost:8001/api/v1/namespaces/kube-system/services/kubernetes-dashboard/proxy/``.
```

Comment:
@ajaeger FYI it is also wrong here https://kubernetes.io/docs/tasks/access-application-cluster/web-ui-dashboard/#command-line-proxy